cli/EAP7-1023_Compact_Echo_DMR.adoc

```diff
@@ -0,0 +1,55 @@
+= EAP7-1023 Compact output for echo-dmr command
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org/browse/
+
+== Overview
+
+The intent of this feature is to introduce a new option to the echo-dmr command
+ in order to display compact (a single line) output. The main use case is the ability, 
+in conjunction with json output, to generate content directly consumable by the 
+HTTP management REST API. 
+
+== Issue Metadata
+
+=== Issue:
+
+* {issue-base-url}EAP7-1023[EAP7-1023]
+
+=== Related Issues:
+
+* {issue-base-url}WFCORE-3933
+
+=== Dev Contacts:
+
+* mailto:jdenise@redhat.com[Jean-François Denise]
+
+=== QE Contacts:
+
+* mailto:eduda@redhat.com[Erich Duda]
+
+=== Affected Projects or Components:
+
+* WildFly Core CLI
+
+
+== Requirements
+
+=== Hard requirements
+
+* Echo-dmr should be able to display its output in a single line.
+* Applies to both JSON and DMR custom syntax.
+
+
+== Test Plan
+
+* New test cases to cover completion and execution will be added to WFCORE repository.
```

Comment:
@jfdenise Since we agreed that tests will be developed by you (dev) can you please be more specific what tests you plan to develop? Thank you.

Reply:
Ok, updated.